hw/ip/aes/data/aes.hjson

```diff
@@ -256,13 +262,15 @@
 # input data registers
     { multireg: {
       name: "DATA_IN",
+      resval: "0",
       desc: '''
         Input Data Registers.
         If MANUAL_OPERATION=0 (see Control Register), the AES unit automatically starts encryption/decryption after these register have been written.
```

Comment:
... after these register have ... Either "after this register has" or mention which other registers are involved, and "these registers".

Reply:
Good point, I've changed that now.